tests/tests.py

```diff
@@ -510,6 +510,76 @@ def test_search_bind_with_bad_credentials(self):
 
         self.assertIsNone(user)
 
```

Comment:
equivalent to:
https://github.com/python-ldap/python-ldap/blob/2fb97af1e025a95d1ea86b9aeac6235358e52278/Tests/t_ldap_sasl.py#L78